pbft/pbft.cpp

```diff
@@ -1610,6 +1707,23 @@ pbft::proposed_config_is_acceptable(std::shared_ptr<pbft_configuration> /* confi
     return true;
 }
 
+bool
+pbft::validate_config_in_newview(const pbft_msg& msg)
```

Comment:
Is this still supposed to be here?

Reply:
It's only used by a new node joining the swarm to do basic validation that the config is accepted by 2f+1 nodes. Again we can't do a full validation due to the trust chicken/egg situation though.